jenkins/branch.name

```diff
@@ -0,0 +1 @@
+b0.70
```

Comment:
This should probably be derived from the VERSION file - but not for this backport.

Reply:
That is arguable, as we might need to make more than one branch for a given release stream to deal with problems and special cases.

Reply:
Filed issue #1966 to track.